client/cypress/plugins.ts

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
Why exactly is this needed? I'm not seeing any non camelCase variables in this file

Reply:
It's the field names on L21-23